docker-compose.yml

```diff
@@ -202,11 +202,11 @@ services:
     restart: always
 
   collabora:
-    image: collabora/code:23.05.10.1.1
+    image: collabora/code:24.04.5.1.1
     command: ['bash', '-c', 'coolconfig generate-proof-key ; /start-collabora-online.sh']
     environment:
       DONT_GEN_SSL_CERT: YES
-      extra_params: --o:ssl.enable=false --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_URL:-https://host.docker.internal:9200}
+      extra_params: --o:ssl.enable=false --o:ssl.ssl_verification=false --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_URL:-https://host.docker.internal:9200}
```

Comment:
Could we use the value from OCIS_INSECURE?

Reply:
I am not sure about this, we would need to negotiate it, and the other extra params, doesn't it add to much confusion ?

Reply:
Checked: it's not easy to negotiate it, we would need to write bash script to the docker-compose, which would lead to spaghetti-code, so sorry but no :( :( :(